recipes-support/provisioning/files/phytec-board-config.sh

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env bash
```

Comment:
Please use the POSIX shell `/bin/sh` instead of BASH. This way, we can remove the RDEPENDS on BASH.

Reply:
is changed